src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.*;
+import java.util.ArrayList;
+
+public class Tagging {
+    private static ArrayList<String> list;
```

Comment:
the variable naming for this is not very clear. `list` is too generic. 

For this implementation this would be what was described in the second diagram of the Learning Outcome. Look around to see how you can do it using the first diagram of the LO. There are some advantages of not putting the list of taggings in this class.